src/content/tools/devtools/release-notes/release-notes-2.37.1-src.md

```diff
@@ -0,0 +1,39 @@
+# DevTools 2.37.1 release notes
+
+The 2.37.1 release of the Dart and Flutter DevTools
+includes the following changes among other general improvements.
+To learn more about DevTools, check out the
+[DevTools overview](/tools/devtools/overview).
+
+## General updates
+
+* Improve messaging when a screen is unavailable for the platform of the
```

Comment:
can you change the tense of all entries to past tense? e.g. Improved, fixed, etc.

https://github.com/flutter/devtools/blob/master/packages/devtools_app/release_notes/README.md#updating-next_release_notesmd

Reply:
Done!